### PR TITLE
extend CONTRIBUTING.md and README.md

### PR DESCRIPTION
**Pull request by BlaiZephyr**
just a small addition because it's easy to miss CONTRIBUTING.md's existence.
## Checklist
- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ Such fix commits should ideally be squashed into one big commit using ``git comm
 
 A lot of the style offenses can be fixed automatically by running the fix script `./scripts/fix_style.py`
 
+Our Script uses Clang-Format version 10.
+if your Package-Manager no longer provides this version, you can download it from [here](https://github.com/muttleyxd/clang-tools-static-binaries/releases/tag/master-5b56bb49)
+
```
Comment (heinrich5991) [inline CONTRIBUTING.md:29]: I think we could instead ask people to download it from pip, that seems more official: Can you try whether that works for you?

https://pypi.org/project/clang-format/10.0.1.1/

File: README.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ You can get binary releases on the [DDNet website](https://ddnet.org/downloads/)
 
 - [Code Browser](https://ddnet.org/codebrowser/DDNet/)
 - [Source Code Documentation](https://codedoc.ddnet.org/) (very incomplete, only a few items are documented)
+- [Contributing Guide](https://github.com/ddnet/ddnet/blob/master/CONTRIBUTING.md)
```
[the remark this answers is not in the document]
Comment (BlaiZephyr) [reply README.md]: forgot that Github is that smart, changed!

File: CONTRIBUTING.md (hunk 1 above)
Comment (BlaiZephyr) [reply CONTRIBUTING.md:28]: sounds good to me, i didn't even think about using pip ^^

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -24,6 +24,9 @@ Such fix commits should ideally be squashed into one big commit using ``git comm
 
 A lot of the style offenses can be fixed automatically by running the fix script `./scripts/fix_style.py`
 
+Our Script uses Clang-Format version 10.
+if your Package-Manager no longer provides this version, you can download it from [here](https://github.com/muttleyxd/clang-tools-static-binaries/releases/tag/master-5b56bb49)
```
[the remark this answers is not in the document]
Comment (BlaiZephyr) [reply CONTRIBUTING.md]: sounds more humane, thanks!

**Comment by heinrich5991** [comment]
Have you checked that getting clang-format from pip works on your machine?

**Comment by BlaiZephyr** [comment]
> Have you checked that getting clang-format from pip works on your machine?

since i'm on NixOS, the steps required to run any binary from pip is a little special, (e.g i need to wrap it in an FHS-environment). but that did work so i assumed it should work for others too.

**Review by heinrich5991 — APPROVED** [review]
Thanks!

**Pull request merged**